static/js/stat_var_hierarchy/stat_var_section_input.tsx

```diff
@@ -34,6 +34,30 @@ import { USA_CHILD_PLACE_TYPES } from "../tools/map/util";
 
 const TOOLTIP_TOP_OFFSET = 10;
 const TOOLTIP_RIGHT_MARGIN = 20;
+const PLACE_TYPE_MAPPING = {
+  EurostatNUTS1: "State",
+  EurostatNUTS2: "State",
```

Comment:
If we want to word it as "State or equivalent ... ", "County or equivalent ... ", then we could consider AdministrativeArea1 -> State and AdministrativeArea2 -> County

Reply:
done